backend/lib/endpoints/posts.js

```diff
@@ -162,8 +166,10 @@ async function routes(app) {
       );
 
       if (postsErr) {
-        req.log.error("Failed requesting posts", { postsErr });
+        req.log.error(postsErr, "Failed requesting posts");
         throw app.httpErrors.internalServerError();
+      } else if (posts === null) {
+        return []
```

Comment:
small nitpick, I think missing a semicolon `;` 
I think normally when committing this should be fixed with ESLint automatically, not sure if it works for you?

Reply:
it's definitely not working for me. I'll double check. Thanks!